utils/encryptionKeys.js

```diff
@@ -1,29 +1,11 @@
-import { generateKeyPair } from 'crypto';
+import * as paillierBigint from 'paillier-bigint';
 
 /**
  * 
- * @returns {Promise<{publicKey: string, privateKey: string}>} Public and private key pair
+ * @returns {Object} A key pair for the Paillier cryptosystem
```

Comment:
Could probably be more specific but I won't hold back the PR because of this tho 😄